hbase-client/src/main/java/org/apache/hadoop/hbase/replication/ReplicationLoadSink.java

```diff
@@ -47,4 +46,42 @@ public long getTimestampStarted() {
   public long getTotalOpsProcessed() {
     return totalOpsProcessed;
   }
+
+  public static ReplicationLoadSinkBuilder newBuilder() {
```

Comment:
I think the above IA.Priavte annotation on the constructor means we do not want end users to create this? So I suggest that we move ReplicationLoadSinkBuilder to a separated file, marked it as IA.Private, and provide a create method for creating the ReplicationLoadSinkBuilder. And the constructor of ReplicationLoadSink could be package private.

Reply:
If this is the case then let's at least add a IA.Private annotation to the newBuilder method and also the ReplicationLoadSinkBuilder class...

Reply:
OK. Changed the annotation in the latest commit.